tools/parameter-curves/lattice-scripts/compare_curves_and_estimator.py

```diff
@@ -1,5 +1,6 @@
 import os
 import sys
+sys.path.insert(1, os.path.join(sys.path[0], '../../../third_party/lattice-estimator'))
```

Comment:
That's seems to be a bit hardcoded to do that is the script and the makefile correctly setup the PYTHON_PATH for calling the scripts in the right way.

Reply:
changed to use makefile only